frontend/src/container/MetricsTable/index.tsx

```diff
@@ -1,9 +1,13 @@
-import Table, { ColumnsType } from 'antd/lib/table';
+/* eslint-disable react/no-unstable-nested-components */
```

Comment:
@srikanthccv can you please remove this eslint

Reply:
I might need help on this. This was showing not to return jsx element from the following filter functions. What are the ways to achieve that.